.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,7 +10,7 @@ assignees: ''
 **Description of problem:**
 
 - [ ] using xknx standalone
-- [ ] using Home-Assistant knx plugin
+- [ ] using Home-Assistant knx plugin (please use the Home Assistant Issue Tracker)
```

Comment:
```suggestion
- [ ] using Home-Assistant KNX integration (please use the Home Assistant Issue Tracker)
```

Reply:
It's fine for me if a HA KNX bug is filed at xknx. Hard to find something at HAs issues again 🙃
If you prefer to have it at HA then just keep the note.

Reply:
done